docs/frontend/liquid-tags.md

```diff
@@ -41,4 +41,5 @@ Here is a bunch of liquid tags supported on DEV:
 {% spotify spotify:episode:5V4XZWqZQJvbddd31n56mf %}
 {% blogcast 1234 %}
 {% kotlin https://pl.kotl.in/owreUFFUG %}
+{% reddit https://www.reddit.com/r/aww/comments/ag3s4b/ive_waited_28_years_to_finally_havr_my_first_pet %}
```

Comment:
Thanks for adding this to the docs! Could you also add an example to the editor guide? You can find the partial file in `app/views/pages/_editor_guide_text.html.erb`.

Reply:
Yes. In fact, I was expecting this to be mentioned before adding another commit to the PR xD

I'll add the example